content/python/concepts/regex/terms/digit-class/digit-class.md

```diff
@@ -1,52 +1,59 @@
 ---
 Title: "\\d"
-Description: 'Represents a character class that includes all digits (0-9). The inverse of this class is designated with a capital, &bsop;D, and will match any character that is not a digit.'
+Description: 'Matches any digit (0-9) in a given pattern.'
 Subjects:
   - 'Computer Science'
   - 'Data Science'
 Tags:
   - 'Strings'
   - 'Regular Expressions'
+  - 'Characters'
 CatalogContent:
   - 'introduction-to-regular-expressions'
-  - 'practical-data-cleaning'
-  - 'language-parsing'
-  - 'text-preprocessing'
+  - 'paths/computer-science'
 ---
 
-Represents a [regular expression](https://www.codecademy.com/resources/docs/general/regular-expressions) character class that includes all digits (0-9). The inverse of this class is designated with a capital, `\D`, and will match any character that is not a digit.
+The `\d` character class matches any digit (0-9) in a given pattern. The inverse of this class is designated with a capital, [`\D`](https://www.codecademy.com/resources/docs/python/regex/non-digit-class), and will match any character that is not a digit.
 
 ## Syntax
 
-```py
-re.search(r'\d', 'string 1', <flags>)
+```pseudo
+a_given_time = r'\d:\d\d'
 ```
 
-## Example 1
+Used by itself, the `\d` class will match a single valid character inside the pattern.
+
+## Example
 
-Match the race times within the text:
+The example below matches the race times within the text with the help of the `\d` character class:
 
 ```py
 import re
 
-race_info = '''The top three race finishers were: Jane Smith (2:14), Sarah Long (2:18) and Suzy Reynolds (2:20).'''
+race_info = 'The top three race finishers were: Jane Smith (2:14), Sarah Long (2:18) and Suzy Reynolds (2:20).'
 
-re.findall(r'\d+:\d+', race_info)
+race_times = re.findall(r'\d+:\d+', race_info)
 # The + symbol indicates one or more of the preceding value
 
-# Output: ['2:14', '2:18', '2:20']
+print(race_times)
+```
+
+The output would look like the following:
+
+```shell
+['2:14', '2:18', '2:20']
 ```
 
-## Example 2
+## Codebyte Example
 
-Match everything except the movie titles:
+The following example will match everything except for the movie titles and the colons `:` in the times:
```

Comment:
> It would be nice to know why these are being separated. Is this an seo move?

Like this entry, the [digit class](https://github.com/Codecademy/docs/pull/133/files) was created back in August as a term under Python: Regex.

> I see that there is a separate page for \D. I think that these characters should be part of the same doc.

Interesting point. I'm going to consider this in my next set of changes.